src/algebra/polynomial/basic.lean

```diff
@@ -0,0 +1,62 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson, Jalex Stark.
+-/
+import data.polynomial
+open polynomial finset
+
+/-!
+# Polynomials
+
+Eventually, much of data/polynomial.lean should land here.
+
+## Main results
+
+We define two `ring_hom`s of type `polynomial R →+* R`,
```

Comment:
Should these be `ring_hom`s or even `alg_hom`s?
In fact, I think you can reuse `aeval` for `eval_ring_hom`, which seems to be more general.

Reply:
You're right. I've replaced both of these new ring_homs with (lemmas about) special cases of aeval.